vsphere/vm-http-server/README.md

```diff
@@ -0,0 +1,14 @@
+# VMSphere
+An example of creating a new VM over a given cluster, then SSHing to it and starting an http server.  
+
+## Configuration
+
+### Environment Variables
+- `SSH_PASS` - the ssh password of the vm template
+- `VMSPHERE_SERVER` - the vCenter server name for vSphere API operations  
```

Comment:
Don't you think it's better to be as a Terraform variables?
https://registry.terraform.io/providers/hashicorp/vsphere/latest/docs#argument-reference

Reply:
Most of these are need outside Terraform as well.  
As for the credentials, I preferred to stick with the way we do it in other templates - via the env variable credentials convention